# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../synaptics/X11R7-lib64.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2005-07-22 00:29:11.000000000 +0200
+++ ./Makefile	2006-03-05 15:29:53.116732250 +0100
@@ -3,8 +3,8 @@
 # Define the TOP variable to build using include files from a local source tree.
 #TOP = /usr/src/redhat/BUILD/XFree86-4.3.0/xc
 
-INSTALLED_X = $(DESTDIR)/usr/X11R6
-LOCAL_X = Xincludes/usr/X11R6
+INSTALLED_X = $(DESTDIR)/usr/X11
+LOCAL_X = Xincludes/usr/X11
 BINDIR = $(DESTDIR)/usr/local/bin
 MANDIR = $(DESTDIR)/usr/local/man
 
@@ -29,7 +29,7 @@
     fi )
   ALLINCLUDES = -I. -I$(X_INCLUDES_ROOT)/include/X11 \
 		-I$(X_INCLUDES_ROOT)/include/X11/extensions \
-		-I$(X_INCLUDES_ROOT)/$(LIBDIR)/Server/include
+		-I$(X_INCLUDES_ROOT)/include/xorg
 else
   SERVERSRC = $(TOP)/programs/Xserver
   ALLINCLUDES = -I. \
@@ -76,7 +76,7 @@
 
 all:: synaptics_drv.o synclient syndaemon
 
-install: $(BINDIR)/synclient $(BINDIR)/syndaemon $(INSTALLED_X)/$(LIBDIR)/modules/input/synaptics_drv.o install-man
+install: $(BINDIR)/synclient $(BINDIR)/syndaemon $(INSTALLED_X)/$(LIBDIR)/xorg/modules/input/synaptics_drv.o install-man
 
 install-man: $(MANDIR)/man1/synclient.1 $(MANDIR)/man1/syndaemon.1 $(MANDIR)/man5/synaptics.5
 
@@ -95,7 +95,7 @@
 $(BINDIR)/syndaemon : syndaemon
 	install -D $< $@
 
-$(INSTALLED_X)/$(LIBDIR)/modules/input/synaptics_drv.o : synaptics_drv.o
+$(INSTALLED_X)/$(LIBDIR)/xorg/modules/input/synaptics_drv.o : synaptics_drv.o
 	install --mode=0644 -D $< $@
 
 synaptics_drv.o: $(OBJS)
